app/views/shared_link_views.ml

```diff
@@ -0,0 +1,94 @@
+open Lib_common
+open Lib_service
+
+let creation_form ~csrf_token =
+  Templates.Util.form
+    ~:Endpoints.Shared_link.create
+    ~csrf_token
+    Tyxml.Html.
+      [ div
+          ~a:[ a_class [ "field columns" ] ]
+          [ p
+              ~a:[ a_class [ "control"; "column"; "is-full" ] ]
+              [ input
+                  ~a:
+                    [ a_input_type `Text
+                    ; a_placeholder "Titre du lien"
+                    ; a_name "link_title"
+                    ; a_class [ "input" ]
+                    ]
+                  ()
+              ]
+          ]
+      ; div
+          ~a:[ a_class [ "field"; "columns" ] ]
+          [ p
+              ~a:[ a_class [ "control"; "column"; "is-full" ] ]
+              [ input
+                  ~a:
+                    [ a_input_type `Url
+                    ; a_placeholder "https://votre-lien.fun"
+                    ; a_name "link_url"
+                    ; a_class [ "input" ]
+                    ]
+                  ()
+              ]
+          ]
+      ; input
+          ~a:
+            [ a_input_type `Submit
+            ; a_value "Soumettre !"
+            ; a_class [ "button"; "is-link" ]
+            ]
+          ()
+      ]
+;;
+
+module List = struct
+  let line shared_link =
+    let open Tyxml.Html in
+    let open Models.Shared_link.Listable in
+    let src =
+      Gravatar.(url ~default:Identicon ~size:48 shared_link.user_email)
+    in
+    let alt = "Avatar of " ^ shared_link.user_name in
+    tr
+      [ td
+          ~a:[ a_class [ "is-vcentered" ] ]
+          [ img ~a:[ a_class [ "image"; "is-48x48" ] ] ~src ~alt () ]
+      ; td
+          ~a:[ a_class [ "is-vcentered"; "is-fullwidth" ] ]
+          [ a ~a:[ a_href shared_link.url ] [ txt shared_link.title ] ]
+      ; td
+          ~a:[ a_class [ "is-vcentered" ] ]
+          [ span
+              [ txt
+                @@ "le "
+                ^ Templates.Util.format_date shared_link.creation_date
```

Comment:
This prints "le 2023-01-13 06:48", and "le 2023-01-13 **à** 06:48" would be better. It's also true for topics and messages, so I think you can change it safely.